### PR TITLE
Orlp 269.fix issues

### PR DESCRIPTION
**Pull request by IhorF**
Fix issues

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java [hunk 1]
```diff
@@ -78,9 +78,9 @@
     @Auditable(action = AuditingAction.DELETE_COMMENT_FOR_COURSE)
     @DeleteMapping(value = "/api/category/{categoryId}/courses/{courseId}/comments/{courseCommentId}")
     @PreAuthorize(value = "@accessToUrlService.hasAccessToDeleteCommentForCourse(#courseCommentId)")
-    public void deleteComment(@PathVariable Long courseCommentId,@PathVariable Long courseId, @PathVariable Long categoryId) {
+    public ResponseEntity deleteComment(@PathVariable Long courseCommentId, @PathVariable Long courseId, @PathVariable Long categoryId) {
         LOGGER.debug("Deleted comment with id:{} for course with id: {}", courseCommentId, courseId);
         courseCommentService.deleteCommentById(courseCommentId);
+        return new ResponseEntity<>(HttpStatus.OK);
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java]: According to REST Best Practices the body of the DELETE method response should not be empty. You can at least return the id of deleted entity, but please check as I'm not sure.

File: src/main/java/com/softserve/academy/spaced/repetition/service/cardLoaders/AnkiCardExtractor.java [hunk 2]
```diff
@@ -14,38 +16,41 @@
...
-    private List <String> extractDateInToList(Connection conn, String query, String columnName) throws SQLException {
-        List <String> list = new ArrayList <String>();
-        Statement statement = null;
-        ResultSet res = null;
-        statement = conn.createStatement();
-        res = statement.executeQuery(query);
-        while (res.next()) {
-            list.add(res.getString(columnName));
+    private List<String> extractDateInToList(Connection conn, String query, String columnName) throws SQLException {
+        List<String> list = new ArrayList<>();
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/service/cardLoaders/AnkiCardExtractor.java]: As you are catching the SQLException the method doesn't throw it any more.

File: src/main/java/com/softserve/academy/spaced/repetition/service/cardLoaders/AnkiCardExtractor.java [hunk 3]
```diff
@@ -54,10 +59,10 @@
         return result;
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/service/cardLoaders/AnkiCardExtractor.java]: The result is always a newly initialized and empty list? What for?

File: src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java (hunk 1 above)
Comment (IhorF) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java]: Most sources say that DELETE must return response with a 204 status and an empty body.

**Pull request merged**